chainer/utils/sparse.py

```diff
@@ -123,6 +149,36 @@ def to_coo(x, ldnz=None, requires_grad=False):
             row[i] = coo.row
             col[i] = coo.col
         shape = x.shape[1:]
-        return CooMatrix(data, row, col, shape, requires_grad)
+        order = None
+        if x.flags['C_CONTIGUOUS']:
```

Comment:
Ditto. Or maybe better if you could unify the return statements for case ndim == 2 and 3. For DRY.